centreon/src/Core/Security/ProviderConfiguration/Domain/OpenId/Model/CustomConfiguration.php

```diff
@@ -33,7 +33,7 @@
 use Core\Security\ProviderConfiguration\Domain\Model\GroupsMapping;
 use TypeError;
 
-final class CustomConfiguration implements CustomConfigurationInterface, OpenIdCustomConfigurationInterface
```

Comment:
Why remove 'final' keyword ?

Reply:
I need to be able to mock the class in the tests